var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -227,6 +228,48 @@ class LlvmAmdgpu(CMakePackage):
         when="@master +rocm-device-libs",
     )
 
+    for d_version, d_shasum in [
+        ("5.6.1", "4de9a57c2092edf9398d671c8a2c60626eb7daf358caf710da70d9c105490221"),
+        ("5.6.0", "30875d440df9d8481ffb24d87755eae20a0efc1114849a72619ea954f1e9206c"),
+    ]:
+        resource(
+            name="hsa-runtime",
+            placement="hsa-runtime",
+            url="https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-{0}.tar.gz".format(
+                d_version
+            ),
```

Comment:
FWIW. We're encouraging use of f-strings in the core so you could switch this here while you're making the change:

```suggestion
            url=g"https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-{d_version}.tar.gz"
```